testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketSslGreetingTest.java

```diff
@@ -118,6 +117,8 @@ public void testSslGreeting() throws Throwable {
     }
 
     public void testSslGreeting(ServerBootstrap sb, Bootstrap cb) throws Throwable {
+        ByteBuf greeting = Unpooled.buffer().writeByte('a');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "greeting". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "greeting" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1481)